docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,178 @@
+1. Tag with the patch release command on the train branch, `./release.sh patch`
+1. Reference everything at and under the instructions beginning with "Do some manual checks" for [a regular release](#a-regular-release). Keep in mind the diff will be against `origin/train-###`, and instead of creating a new Bugzilla deploy bug, you'll copy the output and paste it in the existing bug.
 
-### The Happy Path: A regular release
+<details>
+    <summary>See a diagram of a patch release with the <code>uplift</code> approach, including what <code>./release.sh patch</code> does</summary>
+    <img src={patchUpliftReleaseDiagram} />
 
-![A simplified merging diagram](../../assets/fxa-release1.png)
+In the scenario above, a regular release happened and `v1.100.0` was tagged and pushed to production.  Later, to fix a regression, a patch was landed *directly on the branch* rather than on `main`.  `release.sh patch` was run and `v1.100.1` was tagged.  Four more commits landed on the branch and `release.sh patch` tagged a `v1.100.2`.  In this scenario there were two patch releases in addition to the regular release at the end of the sprint.
+</details>
 
-During a regular release, running `release.sh` will create an appropriately named branch, update a few files like the Changelogs, and create a tag.  The Release Owner will push the branch to github and open a pull request back to `main`.
+An example of commands to run for this kind of patch release are:
 
-An example of commands to run for a release are:
 ```bash
 git checkout main
 git pull
-./release.sh
+git checkout train-100
+git pull
+git branch -b train-100-uplift-1
+git cherry-pick <commit1> # do these one at a time to ensure there's no merge conflicts
+git cherry-pick <commit2>
+(repeat as needed)
+git log  # Verify you picked what you wanted to
+git push origin train-100-uplift-1
+# On GitHub:
+# 1) make a Pull Request for train-100-uplift-1 to merge into train-100
+# 2) Ask for review
+# 3) Wait for CI to pass ✅
+# 4) Merge the Pull Request
+git checkout train-100
+git pull
+./release.sh patch
 # Follow the instructions printed
 ```
 
-### A Patch Release
+#### Alternate way 1, the YOLO way, without an `-uplift` branch
+
+Alternatively, you don't _have_ to create an `-uplift` branch to do a patch release. If you feel exceptionally confident in the release, you can choose to directly `cherry-pick` commits onto the `train-###` branch and then tag.
+
+The downside to this approach is that a teammate can't review what you've cherry-picked, and if CI fails after you've tagged, you might have to `cherry-pick` in an additional missed fix and then tag the next minor release. You may also push the train branch with the cherry-pick but _prior to tagging_ if you want to see CI run first.
 
-![A patch release diagram](../../assets/fxa-release4.png)
+#### Alternate way 2, the direct PR way
```

Comment:
I'm kinda hesitant to even add alternatives to docs, but I think it's just a personal preference.  No biggie leaving it in